Contentstack.Core.Tests/QueryTest.cs

```diff
@@ -70,6 +70,24 @@ public async Task FetchEntriesPublishFallback()
             }
         }
 
+        [Fact]
+        public async Task FetchEntriesPublishMetadata()
+        {
+            List<string> list = new List<string>();
+            list.Add("en-us");
+            list.Add("ja-jp");
+            ContentstackCollection<Entry> entries = await client.ContentType(source).Query()
+                .SetLocale("ja-jp")
+                .IncludeMetadata()
+                .Find<Entry>();
+            ;
+            Assert.True(entries.Items.Count() > 0);
```

Comment:
The assertion should verify the URL parameters. It should consider the tests passed only if the parameter "include_metadata" is included in the URL.